api/Document.json

```diff
@@ -11519,9 +11513,15 @@
                 "prefix": "webkit"
               }
             ],
-            "chrome_android": {
-              "version_added": "33"
-            },
+            "chrome_android": [
+              {
+                "version_added": "33"
+              },
+              {
+                "version_added": "18",
+                "prefix": "webkit"
```

Comment:
Yep, seems like the prefixed "webkitvisibilitychange" event is still being fired:
https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/core/dom/document.cc;l=1932;drc=df872ce8fcce25af51aa6b0f9fe8b1135b687524